pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php

```diff
@@ -821,7 +821,7 @@ protected function get_pickup_request_args() {
 			}
 
 			$args['dhl_settings']['shipper_country'] = PR_DHL()->get_base_country();
-			$args['dhl_settings']['participation']   = $this->shipping_dhl_settings[ 'dhl_participation_' . $dhl_label_items['pr_dhl_product'] ];
+			$args['dhl_settings']['participation']   = ! empty( $dhl_label_items ) ? $this->shipping_dhl_settings[ 'dhl_participation_' . $dhl_label_items['pr_dhl_product'] ] : '';
```

Comment:
<img width="399" alt="image" src="https://github.com/user-attachments/assets/8ffac906-d5a6-4739-a5ba-fcdd373b18f7" />
`$dhl_label_items` **is undefined**